src/test/java/doctor/doctor1.java

```diff
@@ -0,0 +1,14 @@
+package doctor;
+
+public class doctor1 {
+	
+	public void login()
+	{
+		System.out.println("helloo..");
```

Comment:
please add new varaible to strore doctor id

Reply:
added code  , please review and let me know if i need any code changes.

Reply:
okay done.please merge the code

Reply:
okay